apps/prairielearn/src/api/v1/endpoints/courseInstanceAccessRules/index.js

```diff
@@ -1,12 +1,12 @@
 // @ts-check
 const asyncHandler = require('express-async-handler');
-const path = require('path');
-const express = require('express');
-const router = express.Router({ mergeParams: true });
+import * as path from 'node:path';
+import * as express from 'express';
 
-const sqldb = require('@prairielearn/postgres');
+import * as sqldb from '@prairielearn/postgres';
```

Comment:
Would it be more idiomatic to `import { loadSql } from ...`?

Reply:
Yeah, it would, I've largely been doing the easy mechanical thing for all these PRs. I can come along later and fix these all in on go pretty easily.

Reply:
Yep, no worries, I'm partly just making sure I understand our patterns here.